src/styles/stepTracker.css

```diff
@@ -0,0 +1,10 @@
+section {
```

Comment:
don't declare a global style for section. use a class name for the step tracker. (Example: What if you use a section in a different component?)